roles/ceph-common/tasks/checks/check_mandatory_vars.yml

```diff
@@ -22,6 +22,21 @@
   tags:
     - package-install
 
+- name: make sure ceph_stable_release is set
+  fail:
+    msg: "make sure ceph_stable_release is set to a release name, http://docs.ceph.com/docs/jewel/releases/"
```

Comment:
Yes but I was trying to avoid re-editing the code for every new release...
I was hoping the official doc could be the source of truth... It looks like it's not the case